examples/stress-nodes.js

```diff
@@ -14,11 +14,9 @@ export default function () {
     .withVM("100")
     .withTimeout("15s")
     .inNodes(nodeNames)
+    .autocleanup(true)
```

Comment:
@dgzlopes This sentence is causing an error: `TypeError: Value is not an object: true` I can't figure out why passing a boolean fails here.

Reply:
I was able to fix it like this :sweat_smile: 

```
export class StressNodesAttack {
    constructor() {
	    this.autocleanup = true
    }

    withAutocleanup(autocleanup) {
	    this.autocleanup = autocleanup
	    return this
    }

    autocleanup(autocleanup) {
	    this.autocleanup = autocleanup
	    return this
    }
}

export default function () {
  const stressNodesAttack = new StressNodesAttack();

  // This works
  stressNodesAttack.withAutocleanup(true);
  console.log("withAutocleanup");

  // This fails
  stressNodesAttack.autocleanup(true);
  console.log("autocleanup");
}
```

Reply:
I suppose it crashes b/c your auto cleanup field, is not private. And it shares the same name as your method.